WssElixir/app/src/main/java/com/example/wsselixir/view/MyInformationActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.wsselixir.view
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.example.wsselixir.databinding.ActivityMyinformationBinding
+
+class MyInformationActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMyinformationBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMyinformationBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        setInformaionIntent()
+
+    }
+
+    private fun setInformaionIntent() {
+        val intent = intent
+        val name = intent.getStringExtra("name")
+        val mbti = intent.getStringExtra("mbti")
+
+        binding.tvMyInfoPutName.text = "$name"
+        binding.tvMyInfoPutMBTI.text = "$mbti"
```

Comment:
```suggestion
        binding.tvMyInfoPutName.text = intent.getStringExtra("name")
        binding.tvMyInfoPutMBTI.text = intent.getStringExtra("mbti")
```
로도 사용 가능합니다! 그리고 intent 변수는 지워도 될 것 같아요

Reply:
가독성이 훨씬 좋아지네요!! 반영해보겠습니다!!